tutorials/tutorial_01_basic_app/src/main.cpp

```diff
@@ -10,9 +10,15 @@ using namespace sl_iot;
 using json = sl_iot::json;
 
 int main(int argc, char **argv) {
-    //Init IoT with the SL_APPLICATION_TOKEN environment variable
+    //In service deployment init IoT with the SL_APPLICATION_TOKEN environment variable
+    //In development you can simply init it with the application name
     const char * application_token = ::getenv("SL_APPLICATION_TOKEN");
-    STATUS_CODE status_iot = IoTCloud::init(application_token);
+    STATUS_CODE status_iot;
+    if (!application_token) {
+        status_iot = IoTCloud::init("basic_app");
```

Comment:
you may include this in the API using pattern if no string is given use basic app

Reply:
Removed all if application_token